src/System.Management.Automation/CoreCLR/CorePsPlatform.cs

```diff
@@ -195,7 +197,7 @@ internal static bool IsInbox
         /// Some common environment variables used in PS have different
         /// names in different OS platforms
         /// </summary>
-        internal static class CommonEnvVariableNames
+        public static class CommonEnvVariableNames
```

Comment:
Actually, perhaps we should keep it internal for now until https://github.com/PowerShell/PowerShell-RFC/blob/master/1-Draft/RFC0019-PowerShell-Core-Interop-Module.md gets resolved